Ports/openssl/patches/ocsp_fd_set.patch

```diff
@@ -0,0 +1,13 @@
+--- a/apps/ocsp.c	2020-04-05 16:49:54.499380971 +0100
++++ b/apps/ocsp.c	2020-04-05 16:50:13.175380426 +0100
+@@ -94,6 +94,10 @@
+ #  endif
+ # endif
+ 
++# if defined(OPENSSL_SYS_SERENITY)
++#  include <fd_set.h>
```

Comment:
Please include `<select.h>` instead. `<fd_set.h>` is a non-standard header that only exists so the kernel can include it and get the same exact `fd_set`-related definitions as userspace code. :)

Reply:
Thanks, fixed this.